compute/client_library/ingredients/disks/сonsistency_groups/clone_disks_consistency_group.py

```diff
@@ -0,0 +1,52 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# This is an ingredient file. It is not meant to be run directly. Check the samples/snippets
+# folder for complete code samples that are ready to be used.
+# Disabling flake8 for the ingredients file, as it would fail F821 - undefined name check.
+# flake8: noqa
+
+from google.cloud import compute_v1
+
+
+# <INGREDIENT consistency_group_clone_disks>
+def clone_disks_to_consistency_group(project_id, group_name, group_region):
+    """
+    Clones disks to a consistency group in the specified region.
+    Args:
+        project_id (str): The ID of the Google Cloud project.
+        group_name (str): The name of the consistency group.
+        group_region (str): The region where the consistency group is located.
```

Comment:
My subjective preference is to have the following order of params: project, region, group_name - like, descending in size stuff :) and matched the order in the URI-style identifiers of groups.